opentelemetry-otlp/src/exporter/mod.rs

```diff
@@ -66,7 +66,7 @@ pub(crate) mod tonic;
 
 /// Configuration for the OTLP exporter.
 #[derive(Debug)]
-pub struct ExportConfig {
+pub struct ExporterConfig {
```

Comment:
also suggest to keep the renaming to a separate, small pr, so it is easier to review.